src/runner/runESLint.js

```diff
@@ -1,6 +1,13 @@
 const { ESLint } = require('eslint');
+const { version } = require('eslint/package.json');
+const findUp = require('find-up');
+const semver = require('semver');
+
 const getESLintOptions = require('../utils/getESLintOptions');
 
+const ESLINT_SUPPORTS_FLAT_CONFIG = semver.satisfies(version, '>=8');
+const ESLINT_FLAT_CONFIG_DEFAULT_FILENAME = 'eslint.config.js';
```

Comment:
i don't think this is needed since it's only used once, and is hardcoded in eslint?

Reply:
Yeah, I don't care too much, my thought process here was just to match the eslint implementation as closely as possible and they have it [extracted out to a const](https://github.com/eslint/eslint/blob/d85efad655deacc0dc3fdbbace33307094c3b91b/lib/eslint/flat-eslint.js#L92) as well but this is not something I have any opinion on and I'm happy to change it however if you care about it.